src/Header/Header.less

```diff
@@ -87,6 +83,9 @@
 		.moon-header-title-below,
 		.moon-header-sub-title-below {
 			line-height: 39px;
+			.enyo-locale-non-latin & {
+				line-height: @moon-non-latin-medium-header-line-height;
```

Comment:
Why set the `@moon-non-latin-medium-header-line-height` to the `.moon-header-title-below` and `.moon-header-sub-title-below` classes?